core/include/optee_rpc_cmd.h

```diff
@@ -148,6 +148,28 @@
  */
 #define OPTEE_RPC_CMD_BENCH_REG		20
 
+/*
+ * Issue master requests (read and write operations) to an I2C chip.
+ *
+ * [in]     value[0].a	    Transfer mode (check enum rpc_i2c_mode for modes).
```

Comment:
This should mention `OPTEE_MSG_RPC_CMD_I2C_TRANSFER_*` defined below instead of `enum rpc_i2c_mode`.